jobs/nightly-samba-rpm-builds.yml

```diff
@@ -41,14 +49,14 @@
         - spuiuk
         - nixpanic
         cron: H/5 * * * *
-        status-context: 'samba-build-rpms/centos{centos_version}'
+        status-context: 'samba-build-rpms/centos{centos_version}-{samba_branch}'
```

Comment:
the status context is what will be displayed as CI job in GitHub pull requests. You may want to do it like `samba-build-rpms/centos8/samba-v4-14-test` instead of `samba-build-rpms/centos8-v4-14-test` as the last is less descriptive?

Reply:
I agree.